haystack/preview/dataclasses/__init__.py

```diff
@@ -1 +1,3 @@
-from haystack.preview.dataclasses.document import Document
+from haystack.preview.dataclasses.document import Document, ContentType
```

Comment:
This is useful, document stores might need it to take decisions when ingesting documents